lib/build.js

```diff
@@ -16,6 +16,22 @@ const uiUri = Symbol('uiUri');
 const ABORT_CODE = 130; // 128 + SIGINT 2 (^C)
 const TEMPORAL_JWT_TIMEOUT = 12 * 60; // 12 hours in minutes
 
+// SCM handles only SUCCESS, PENDING & FAILED status
+const SCM_STATE_MAP = {
```

Comment:
BLOCKED: 'PENDING',
COLLAPSED: '',
 FROZEN: ''
  FIXED: '',
  ones empty should not be handled

Reply:
Made the changes accordingly.